ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -44,23 +44,35 @@
     CLI
     <Hds::Badge @text="kv get" @color="neutral" />
   </h3>
-  <p class="helper-text has-text-grey has-bottom-padding-s">
+  <p class="helper-text has-text-grey-light has-bottom-padding-s">
     This command retrieves the value from KV secrets engine at the given key name. For other CLI commands,
     <DocLink @path="/vault/docs/commands/kv">
       learn more.
     </DocLink>
   </p>
-  <CodeSnippet data-test-commands="cli" @codeBlock={{this.commands.cli}} />
+  <Hds::CodeBlock
```

Comment:
Can we delete `CodeSnippet` now that HDS has a component to replace it?

Also, if you want to handle cleanup + removal in a separate PR that makes sense to me as well

Reply:
Ahh - I'm noticing there still are some CodeSnippet instances - did those pass ally testing? Seems like we should swap those soon so all code blocks look the same 🤔

Reply:
Yeah, my intention is to do more cleanup and removal separately. I didn't want the sidebranch PR to blow too much out of scope

Reply:
Sounds good! Cleanup is just easy to lose track of and I wanted to make sure we did it while we still had context that it was okay to delete things.

Reply:
Actually there weren't that many instances so I went ahead and removed/replaced CodeSnippets in this PR 👍